### ISSUE TITLE
Include_class availability, as in yolov5, to filter labels?

### ISSUE THREAD

**Issue by ilpapds** [ultralytics/ultralytics#1294]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Is there a include_class availability, as in yolov5, to filter labelsfor training on custom data?

**Comment by AyushExel** [comment]
This should now be available in the latest release by @Laughing-q

**Comment by Laughing-q** [comment]
@ilpapds 
usage:
```bash
yolo detect train classes=0  # classes=0,1,... for multiple classes
```
```python
model.train(classes=0)   # classes=[0,1,2...,...] for multiple classes
```
